engine/samples/hello-cubos/main.cpp

```diff
@@ -0,0 +1,45 @@
+#include <cubos/engine/cubos.hpp>
```

Comment:
```suggestion
#include <cubos/engine/cubos.hpp>

using cubos::engine::Cubos;
```

I would include the `using` statement here - it would also be nice to show this initial include + using snippet in the guide, since its the first sample the reader will see.